src/private/WRLDPointOnPath.mm

```diff
@@ -109,7 +109,9 @@ - (WRLDPointOnRouteResult*) makeWRLDPointOnRouteFromPlatform:(Eegeo::Routes::Poi
     
     WRLDRouteStep *routeStep = [routeSection.steps objectAtIndex:pointOnRouteInfo.GetRouteStepIndex()];
     
-    WRLDPointOnRouteResult* newPointOnRouteInfo = [[WRLDPointOnRouteResult alloc] initWithResultPoint:resultPoint inputPoint:inputPoint distanceFromInputPoint:distanceFromInputPoint fractionAlongRoute:fractionAlongRoute fractionAlongRouteSection:fractionAlongRouteSection fractionAlongRouteStep:fractionAlongRouteStep routeStep:routeStep routeSection:routeSection];
+    int pathSegmentStartVertexIndex = pointOnRouteInfo.GetPointOnPathForClosestRouteStep().GetIndexOfPathSegmentStartVertex();
+    
+    WRLDPointOnRouteResult* newPointOnRouteInfo = [[WRLDPointOnRouteResult alloc] initWithResultPoint:resultPoint inputPoint:inputPoint distanceFromInputPoint:distanceFromInputPoint fractionAlongRoute:fractionAlongRoute fractionAlongRouteSection:fractionAlongRouteSection fractionAlongRouteStep:fractionAlongRouteStep routeStep:routeStep routeSection:routeSection routeSectionIndex:pointOnRouteInfo.GetRouteSectionIndex() routeStepIndex:pointOnRouteInfo.GetRouteStepIndex() pathSegmentStartVertexIndex:pathSegmentStartVertexIndex];
```

Comment:
Fix indentation for all params

Reply:
Have a look if we can avoid information being passed such as 'routeSection' and 'routeSectionIndex'.